Goldmine/Gold.java

```diff
@@ -0,0 +1,71 @@
+
+
+import java.util.*; 
+  
+class Gold{ 
+      
+    static final int MAX = 100; 
+      
+    static int getMaxGold(int gold[][],  
+                              int m, int n) 
+    { 
+          
+        int goldTable[][] = new int[m][n]; 
+          
+        for(int[] rows:goldTable) 
+            Arrays.fill(rows, 0); 
+      
+        for (int col = n-1; col >= 0; col--) 
+        { 
+            for (int row = 0; row < m; row++) 
+            { 
+                  
+                int right = (col == n-1) ? 0 
```

Comment:
This ternary operator statement can be written in a single line. Similarly for subsequent statements.